BaseUnit.cpp

```diff
@@ -19,6 +21,83 @@ std::string BaseUnit::showStats() const
 {
 	return Name + ": HP:" + std::to_string(HP) + " DMG: " + std::to_string(DMG);
 }
+BaseUnit BaseUnit::parseUnit(const std::string& file_name)
+{
+	std::fstream infile(file_name);
+
+	if (!infile.is_open())
+	{
+		throw(file_name);
```

Comment:
Jobb lenne erre definiálni egy exception típust, mint a stringet dobni. Vagy akár az `std::runtime_error`t is lehet használni.